docs/modules/spark-k8s/partials/supported-versions.adoc

```diff
@@ -3,6 +3,7 @@
 // Stackable Platform documentation.
 // Please sort the versions in descending order (newest first)
 
-- 3.5.0 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 11)
+- 3.5.1 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 11)
```

Comment:
I guess we want to bump Hadoop to 3.3.6 here?

Reply:
Good point, I was thinking the same :+1: 
TLDR: @maltesander decided in https://github.com/stackabletech/docker-images/commit/b407c0cbf4f4e1f98b3c009363928144e25a1d8e to not bump it, I'm happy to stick to that and bump it in the next release.

Also our somewhat recommended product version (we don't have that yet officially) is probably 3.3.4, as 3.3.6 [has problems](https://github.com/stackabletech/hdfs-operator/issues/440), so I guess this would be the matching HDFS client version. But all of this are no strong opinions, I just have the feeling the larger portion of our developers would prefer to not change it just days before the release.

But we can bring this up during daily

Reply:
Yeah no strong opinion from my side, just that we did not figure out (yet?) what is not working with 3.3.6. for some.